bot/cogs/tags.py

```diff
@@ -28,21 +29,31 @@ class Tags(Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
         self.tag_cooldowns = {}
-
         self._cache = {}
-        self._last_fetch: float = 0.0
 
-    async def _get_tags(self, is_forced: bool = False) -> None:
+    @Cog.listener()
+    async def on_ready(self) -> None:
+        """Runs the code before the bot has connected."""
+        await self.get_tags()
+
+    async def get_tags(self) -> None:
         """Get all tags."""
-        # refresh only when there's a more than 5m gap from last call.
-        time_now: float = time.time()
-        if is_forced or not self._last_fetch or time_now - self._last_fetch > 5 * 60:
-            tags = await self.bot.api_client.get('bot/tags')
-            self._cache = {tag['title'].lower(): tag for tag in tags}
-            self._last_fetch = time_now
+        # Save all tags in memory.
+        tag_files = os.listdir("bot/resources/tags")
+        for file in tag_files:
+            p = Path("bot", "resources", "tags", file)
+            tag_title = os.path.splitext(file)[0].lower()
```

Comment:
Don't use pathlib for half the problems and `os.path` for the other. You only need one of these.

If you want the title of the file without the extension for a Path, just use the `.stem` property of a `Path` object.